README.md

```diff
@@ -3,6 +3,8 @@ CMP is a tool for publishers to engage users of their properties and gather & st
 
 ### Installation
 
+**Notice:** Installation of the node modules requires a Node version between 4 and 10.
```

Comment:
When I try to run `yarn install` with node 10 I get the following error:
```
± yarn install
yarn install v1.13.0
(node:2039) ExperimentalWarning: The fs.promises API is experimental
[1/4] Resolving packages...
[2/4] Fetching packages...
info fsevents@1.1.3: The platform "linux" is incompatible with this module.
info "fsevents@1.1.3" is an optional dependency and failed compatibility check. Excluding it from installation.
error upath@1.0.2: The engine "node" is incompatible with this module. Expected version ">=4 <=9". Got "10.1.0"
error Found incompatible module
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
```

When I switch to Node 9 I am able to run `yarn install`